integration/renamecollection_compat_test.go

```diff
@@ -55,7 +55,7 @@ func TestRenameCollectionCompat(t *testing.T) {
 			targetNSFrom: targetDB.Name() + "." + targetCollection.Name(),
 			compatNSFrom: compatDB.Name() + "." + compatCollection.Name(),
 			targetNSTo:   targetDB.Name() + ".newCollection",
-			compatNSTo:   targetDB.Name() + ".newCollection",
+			compatNSTo:   compatDB.Name() + ".newCollection",
```

Comment:
Aren't `targetDB.Name()` and `compatDB.Name()` the same? Could we `require.Equal(t, compatDB.Name(), targetDB.Name())` and then use two fields (`to` and `from`) instead of four?

Reply:
True, maybe actually I'm on wrong lead.

Reply:
Yeah I'm on the wrong lead, the issue is essentially other test has renamed the collection. so the collection didn't exist anymore.

Reply:
We still could clean-up the test in this PR. Also, let's replace those `any`s with `string`s. Maybe clean-up something else.

Reply:
I think those `any` are testing for invalid bson input. So want to keep them for testing.